tests/python/PyDocStyleBearTest.py

```diff
@@ -23,7 +23,44 @@ def hello():
 
 '''.splitlines(keepends=True)
 
-PyDocStyleBearTest = verify_local_bear(PyDocStyleBear,
-                                       valid_files=(good_file,),
-                                       invalid_files=(bad_file,),
-                                       tempfile_kwargs={"suffix": ".py"})
+PyDocStyleBearTest = verify_local_bear(
+    PyDocStyleBear,
+    valid_files=(good_file,),
+    invalid_files=(bad_file,),
+    tempfile_kwargs={"suffix": ".py"})
```

Comment:
@tushar-rishav The `PyDocStyleBear` won't work if the extension is not `.py`. The `pydocstyle` tool works only if the extension is `.py`.